src/backends/tensorflow/autofill.cc

```diff
@@ -330,6 +330,11 @@ AutoFillGraphDef::Create(
   // attempt to determine the platform. For now we allow multiple versions
   // and only inspect the first verison directory to ensure it is valid.
   // We can add more aggressive checks later.
+  if (version_dirs.size() == 0) {
+      return Status(
+          RequestStatusCode::INTERNAL, "unable to autofill for '" + model_name +
```

Comment:
I think you need to run the formatter on this file

Reply:
Done